lib/zold/version_file.rb

```diff
@@ -9,11 +9,26 @@ def initialize(path, log: Log::Verbose.new)
       @log = log
     end
 
-    # @todo #285:30min Replace this stub with functionality.
-    #  We need to run the script (`yield`) if the version of
-    #  the script is between the saved version and the current one.
     def apply(version)
-      @log.info("Version #{version} doesn't need to be applied.")
+      return unless Gem::Version.new(version) > current_data_version
+
```

Comment:
@ledestin keep this in mind: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html